src/x/instrument/config.go

```diff
@@ -25,6 +25,7 @@ import (
 	"io"
 	"time"
 
+	promfork "github.com/m3db/prometheus_client_golang/prometheus"
```

Comment:
super nit: Maybe just call it `prom` instead of `promfork`?  I know it is a fork, but the package and the module is still prom essentially.